loadtest/control/simplecontroller/controller.go

```diff
@@ -154,7 +155,14 @@ func (c *SimpleController) createActions(definitions []actionDefinition) error {
 			c.connect()
 			return resp
 		},
-		"Logout":   control.Logout,
+		"Logout": func(u user.User) control.UserActionResponse {
+			resp := control.Logout(u)
+			if resp.Err != nil {
+				return resp
+			}
+			u.ClearUserData()
+			return resp
+		},
 		"EditPost": control.EditPost,
 		"Reload": func(u user.User) control.UserActionResponse {
 			return c.reload(false)
```

Comment:
Can we also fix this since we are here? It should be called with `true`.

Reply:
Sure